modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementSuccessTest.java

```diff
@@ -258,7 +249,44 @@ public void testGetOrganization() {
                 .body("id", equalTo(organizationID));
     }
 
-    @Test(dependsOnMethods = "testGetOrganization")
+    @DataProvider(name = "filterQueries")
+    public Object[][] createFilterQueries() {
+        return new Object[][] {
+                {"?filter=name co G&limit=10&recursive=false", false, false},
+                {"?filter=attributes.Country co S&limit=10&recursive=false", true, false},
+                {"?filter=attributes.Country eq Sri Lanka&limit=10&recursive=false", true, false},
+                {"?filter=attributes.Country eq USA&limit=10&recursive=false", false, true}
+        };
```

Comment:
Shall we and one other case to check 
filter=attributes.Country eq USA and name co `<something>`

Reply:
Added a new test case for multiple attribute filtering. 

`?filter=attributes.Country eq Sri Lanka and name co Greater`